frontend/app.scss

```diff
@@ -215,3 +215,7 @@ body > footer {
 .datagrid {
   padding: 0;
 }
+
```

Comment:
tohle se pouzije kde?

Reply:
ten help-block nebo datagrid?

help-block obaluje ty chybový hlášky ve formuláři, když to tam není, tak se chyba nevypíše na novém řádku, ale hned za posledním checkboxem

u datagridů odebírám okraj, kterej mají běžně asi 10px, takže to pak není stejně široký jak obsah okolo